test/tests_quadratic_program.py

```diff
@@ -34,7 +34,7 @@ def __init__(self, testname, universe = 'msci', solver_name = 'cvxopt'):
         super().__init__(testname)
         self._universe = universe
         self._solver_name = solver_name
-        self.data = load_data_msci(os.path.join(os.getcwd(), 'test', f'test_data{os.sep}'))
+        self.data = load_data_msci(os.path.join(os.getcwd(), f'data{os.sep}'))
 
```

Comment:
Does this PR also deletes the folder `test_data`?

Reply:
Yes, you can verify in the fork's branch https://github.com/vfisikop/GeoFin/tree/fix/duplicate_data_files/test